set.mm

```diff
@@ -268475,6 +268517,35 @@ Additional definitions for (multivariate) polynomials
       UHULUIVJIUMQZUNUOUPQPVCUQQGURPBUSZUTAVPVDVQBIVAVQVEVBVFVG $.
   $}
 
+  ${
+    $d ph d y $.  $d C d y $.  $d A d $.  $d R d h y $.  $d I d h y $.
+    $d V k $.  $d I k $.  $d V y $.  $d K y $.
+    mhpsclcl.h $e |- H = ( I mHomP R ) $.
+    mhpsclcl.p $e |- P = ( I mPoly R ) $.
+    mhpsclcl.a $e |- A = ( algSc ` P ) $.
+    mhpsclcl.k $e |- K = ( Base ` R ) $.
+    mhpsclcl.i $e |- ( ph -> I e. V ) $.
+    mhpsclcl.r $e |- ( ph -> R e. Ring ) $.
+    mhpsclcl.c $e |- ( ph -> C e. K ) $.
+    $( A scalar (or constant) polynomial has degree 0.  Compare ~ deg1scl .
```

Comment:
A **nonzero** constant polynomial...

Reply:
the zero polynomial also has degree zero, it just also has all other degrees

Reply:
Yes, and indeed you wrote that above. But as you wrote, this is due to our special convention: a reader who lands on this page might be a bit puzzled. So that first sentence without "nonzero" sounds a bit strange, even if our special definition makes it work also for the zero polynomial.